### PR TITLE
docs(pwa): update firebase instructions for deploying pwa

### PR DESCRIPTION
**Pull request by sean-perkins**
Updates the instructions for using Firebase to deploy a PWA.

Angular: https://ionic-docs-git-docs-update-react-pwa-ionic1.vercel.app/docs/angular/pwa#deploying
React: https://ionic-docs-git-docs-update-react-pwa-ionic1.vercel.app/docs/react/pwa#deploying
Vue: https://ionic-docs-git-docs-update-react-pwa-ionic1.vercel.app/docs/vue/pwa#deploying

Co-authored-by credit has been applied from this PR: https://github.com/ionic-team/ionic-docs/pull/2708.

Note: This is from a community PR, applied to the other similar guides.

### CODE REVIEW

File: docs/angular/pwa.md [hunk 1]
```diff
@@ -82,19 +82,33 @@ If it's the first time you use firebase-tools, login to your Google account with
...
-Answering these next two questions will ensure that routing, hard reload, and deep linking work in the app:
+Answering this next question will ensure that routing, hard reload, and deep linking work in the app:
 :::
 
 **Configure as a single-page app (rewrite all urls to /index.html)?"** Enter "Yes".
 
-**"File www/index.html already exists. Overwrite?"** Enter "No".
+**"File build/index.html already exists. Overwrite?"** Enter "No".
+
+**Set up automatic builds and deploys with github?** Enter "Yes".
```
Comment (thetaPC) [inline docs/angular/pwa.md:101]: Change `github` to `GitHub` for consistency.

File: docs/react/pwa.md [hunk 2]
```diff
@@ -102,22 +99,40 @@ Next, in a Terminal, install the Firebase CLI:
...
 :::note
-Answering these next two questions will ensure that routing, hard reload, and deep linking work in the app:
+Answering this next question will ensure that routing, hard reload, and deep linking work in the app:
 :::
 
 **Configure as a single-page app (rewrite all urls to /index.html)?"** Enter "Yes".
 
 **"File build/index.html already exists. Overwrite?"** Enter "No".
 
+**Set up automatic builds and deploys with github?** Enter "Yes".
```
Comment (thetaPC) [inline docs/react/pwa.md:124]: Change `github` to `GitHub` for consistency.

File: docs/vue/pwa.md [hunk 3]
```diff
@@ -167,21 +167,39 @@ Next, in a Terminal, install the Firebase CLI:
...
-Answering these next two questions will ensure that routing, hard reload, and deep linking work in the app:
+Answering this next question will ensure that routing, hard reload, and deep linking work in the app:
 :::
 
 **Configure as a single-page app (rewrite all urls to /index.html)?"** Enter "Yes".
 
-**"File dist/index.html already exists. Overwrite?"** Enter "No".
+**"File build/index.html already exists. Overwrite?"** Enter "No".
+
+**Set up automatic builds and deploys with github?** Enter "Yes".
```
Comment (thetaPC) [inline docs/vue/pwa.md:192]: Change `github` to `GitHub` for consistency.

**Review by thetaPC — CHANGES_REQUESTED** [review]
I noticed that there was inconsistencies that some questions have quotes while other don't. Is that coming from the Firebase CLI?

**Comment by sean-perkins** [comment]
The questions with quotes are when the CLI provides an option for selection. Otherwise we specify the manual input that the user needs to provide (branch name, project name, etc.). 

I'm open to ideas on how to better format this. I mainly kept what already existed in the docs and added the new information from the community PR.

File: docs/angular/pwa.md (hunk 1 above)
Comment (thetaPC) [inline docs/angular/pwa.md:101]: It seems that this `github` got missed. It should be `Github`.

**Review by thetaPC — CHANGES_REQUESTED** [review]

**Comment by thetaPC** [comment]
> The questions with quotes are when the CLI provides an option for selection. Otherwise we specify the manual input that the user needs to provide (branch name, project name, etc.).
> 
> I'm open to ideas on how to better format this. I mainly kept what already existed in the docs and added the new information from the community PR.

Since it's from the CLI, then let's leave it as is. It would reduce the potential confusion by keeping it exactly the same.

**Review by thetaPC — APPROVED** [review]
LGTM

**Pull request merged**